packages/styles/src/sass/components/header/_universal.scss

```diff
@@ -1,61 +1,69 @@
 @use '../../base/colors' as *;
 @use '../../typography/fonts' as *;
+@use '../../grid/breakpoints' as *;
+@use '../../tokens/font' as *;
 
-.cbp-header {
+.cbp-universal-header {
   align-items: center;
   background-color: $dhs-blue;
   box-sizing: border-box;
   display: flex;
+  flex-wrap: wrap;
   height: 60px;
+  justify-content: space-between;
+  padding-left: 12px;
+  padding-right: 12px;
   top: 0;
   transition: top 500ms ease-in-out;
   width: 100%;
-  
-  & > div {
-    align-items: inherit;
-    display: inherit;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    width: inherit;
-  }
 }
 
-.cbp-header__brand {
+.cbp-universal-header__brand {
   align-items: center;
   display: flex;
-  & > img {
-    height: 44px;
-    width: 44px;
-    margin-right: 8px;
-  }
 
-  & > span {
-    @extend .cbp-heading-xl;
+  &::after {
+    @extend .cbp-heading-l;
+    content: "CBP";
     color: $cbp-text-lightest;
+    margin-left: 8px;
   }
 }
 
-.cbp-header__info {
+.cbp-universal-header__list {
   display: flex;
+
+  & > li {
+    margin-right: 16px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
 }
 
-.cbp-header__item {
+// TODO: Add Secondary Ghost Button Mixin
+.cbp-universal-header__item {
   align-items: center;
   background-color: transparent;
   box-sizing: border-box;
   border: 0;
   border-radius: 3px;
   display: flex;
   color: $cbp-ui-neutral-lighter;
-  font-size: 14px;
+  cursor: pointer;
+  font-size: $cbp-text-button;
   font-weight: 500;
   height: 36px;
   justify-content: center;
-  margin-right: 16px;
   text-decoration: none;
   text-transform: uppercase;
   width: 36px;
 
+  & span {
```

Comment:
This is screaming for utility classes for responsive show/hide/visibly-hidden. We shouldn't have to write this into the component as a one-off and it should be visibly hidden at the small size so that the text can still label the link. Without it, it's not accessible. This is a TODO since we don't have those utility classes yet.